extensions/amp-video/0.1/amp-video.js

```diff
@@ -597,6 +597,23 @@ class AmpVideo extends AMP.BaseElement {
       this.element.dispatchCustomEvent(mutedOrUnmutedEvent(this.muted_));
     });
 
+    // If managed by pool, let it handle the src changes and loading.
+    // Once the duration has changed, then we safely send the Load event.
+    if (this.isManagedByPool_()) {
+      const durationChangedEventUnlisten = listen(
+        video,
+        'durationchange',
+        (event) => {
+          const {target} = event;
+          if (isNaN(target.duration)) {
+            return;
+          }
+          this.element.dispatchCustomEvent(VideoEvents.LOAD);
```

Comment:
Looking at the code path, it looks like it's possible that this event is triggered multiple times (twice?), depending on a race condition between loading the original source and then loading the sources from the media pool.

Would this be a problem?